systemtest/src/main/java/io/strimzi/systemtest/upgrade/CommonVersionModificationData.java

```diff
@@ -82,6 +85,30 @@ public void setProcedures(UpgradeKafkaVersion procedures) {
         this.procedures = procedures;
     }
 
+    public void setFilePaths(Map<String, String> filePaths) {
+        this.filePaths = filePaths;
+    }
+
+    public Map<String, String> getFilePaths() {
+        return filePaths;
+    }
+
+    public String getKafkaFilePathBefore() {
+        return getFilePaths().get("kafkaBefore");
```

Comment:
Not scope of this PR but maybe we should introduce constraints to map the upgrade file attributes? :)

Reply:
Like not null, size etc?
sure we can